internal/util/cmdutil/cmdutil.go

```diff
@@ -82,3 +83,16 @@ func ResolveAbsAndRelPaths(path string) (string, string, error) {
 
 	return relPath, absPath, nil
 }
+
+// DockerCmdAvailable runs `docker ps` to check that the docker command is
+// available, and returns an error with installation instructions if it is not
+func DockerCmdAvailable() error {
+	_, err := exec.Command("docker", "ps").Output()
+	if err != nil {
+		return fmt.Errorf("docker is required to run this command\n" +
```

Comment:
`docker ps` runs with an error doesn't necessarily show that there is no docker. It's also possible that the docker daemon is not running. It will be good to wrap the stderr from the command.

Reply:
done, wrapped stderr for such a case

Reply:
Daemon us not running is not the only possible reason. Some users may have docker run in root privilege. I am thinking add the error message before the installation directions no matter what the error is.

Reply:
updated